### PR TITLE
#442 part3 review home styles

### PR DESCRIPTION
**Pull request by nmadruga**
Extra PR for #442 - part 3

~~### Based on PR
#565~~ MERGED

### Changes
- Add `Stage` component (and temporary style for the stage Label) - to be also used in PR #572 (Application list styles + actions)
- Adding styles to `ReviewHome` and `AssignmentSectionRow`

### CODE REVIEW

**Comment by nmadruga** [comment]
Hi @CarlosNZ and @andreievg can some of you have a look on this PR please? It's 7 days old now.

File: src/containers/Review/ReviewHome.tsx [hunk 1]
```diff
@@ -159,16 +182,63 @@ const getReviewerOptions = (assignments: AssignmentDetails[], currentUserId: num
     },
   ]
   assignments.forEach(({ reviewer: { id, firstName, lastName } }) => {
-    if (!id || !firstName || !lastName) return
+    if (!id || !firstName) return
```
Comment (CarlosNZ) [inline src/containers/Review/ReviewHome.tsx:185]: I think `username` is a more reliable thing to check than `firstName`.

**Review by CarlosNZ — APPROVED** [review]
Just one tiny comment, if you think it needs addressing.

I haven't looked to closely at the styles, as they'll be getting looked at more thoroughly later.

File: src/containers/Review/ReviewHome.tsx (hunk 1 above)
Comment (nmadruga) [reply src/containers/Review/ReviewHome.tsx:185]: No problem. Changing to `username` for now.

**Comment by nmadruga** [comment]
Thanks for the review Carl. Merging this now

**Pull request merged**